java/code_to_gast/java_router.py

```diff
@@ -30,6 +31,11 @@ def node_to_gast(node):
         return node_to_gast(node.expression)
     elif type(node) == list:
         return java_helpers.node_list_to_gast_list(node)
+    elif type(node) == javalang.tree.LocalVariableDeclaration:
+        # our current gAST doesn't support multiple declarations/unsure what that is
+        return node_to_gast(node.declarators[0])
```

Comment:
Don't love the idea of having to call the first item of the list. Feel like when we have done this in the past, we later realized that the other items in the list were actually important and necessary. May be necessary in this case but just something to be mindful of

Reply:
I'll look more into this. BTW this is still being done on the python and js side where we only look at target[0] or declaration[0]. I'll leave the comment in

Reply:
Adding an error check here to return unsupported if more than one elm in list